MOF_Force_Field/dataloader.py

```diff
@@ -0,0 +1,90 @@
+from torch_geometric.data import DataLoader, Data
+import torch
+from torch_geometric.data import InMemoryDataset
+from tqdm import tqdm
+import math
+import matplotlib.pyplot as plt
+import numpy as np
+from sklearn.preprocessing import LabelEncoder
+import pandas as pd
+
+df = pd.read_csv('FIGXAU_V2.csv',skiprows=1, low_memory=False)

```

Comment:
I don't know exactly what you mean by arbitrary, is it a different way to code it with pandas or do you mean just making the name something like "input.csv" instead of FIGXAU_V2.csv?

Reply:
You shouldn't hard code the name of the file, it should be encased in a function that lets you put in the name of the file.